### PR TITLE
{bio}[foss/2022a] PIRATE v1.0.5

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Depends on:
* [x] #17273 
* [x] #17268

### CODE REVIEW

File: easybuild/easyconfigs/p/PIRATE/PIRATE-1.0.5-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'Tarball'
+
+name = 'PIRATE'
+version = '1.0.5'
```
Comment (jfgrimm) [inline easybuild/easyconfigs/p/PIRATE/PIRATE-1.0.5-foss-2022a.eb:4]: this should carry the R versionsuffix too
```suggestion
version = '1.0.5'
versionsuffix = '-R-%(rver)s'
```

Comment (lexming) [reply easybuild/easyconfigs/p/PIRATE/PIRATE-1.0.5-foss-2022a.eb]: fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/17275/commits/d00bd2f451a8fc7317acc07a399f62a6ffa4f803

**Comment by lexming** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 10 out of 10 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/11264961353541045a0c35134f65075d for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**